### PR TITLE
Add new logged_in_user helper

### LINKED ISSUE #8: Logged in user tag
Could we get [this](https://github.com/ausaccessfed/identity-enhancement-service/blob/master/app/views/layouts/application.html.erb#L64-69) turned into a tag within lipstick?

Save having it in every app.

### PR DESCRIPTION
**Pull request by smangelsdorf**
Fixes #8

### CODE REVIEW

File: doc/layout_helpers.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ Creates the application footer using the provided content.
 ```
 </span>
 
+#### `logged_in_user`
+
+Adds text showing which user is currently authenticated. Requires a
+[Gumboot](https://github.com/ausaccessfed/aaf-gumboot)-compliant Subject model.
```
Comment (rianniello) [inline doc/layout_helpers.md]: ```logged_in_user just``` requires an object with ```name``` and ```targeted_id``` right? 

Is adhering to all the rules here https://github.com/ausaccessfed/aaf-gumboot/blob/develop/lib/gumboot/shared_examples/subjects.rb overkill? 

I don't really have a big problem with this. I don't think it needs to be fixed... more of an observation.

Comment (smangelsdorf) [inline doc/layout_helpers.md]: Currently, yes, it only requires an object with `name` and `targeted_id`.

It's overkill, but it's also a useful constraint to have if (for example) we want to change the format later to include email address and/or shared token.

Maybe a better wording would be: "Expects the Subject model to have all attributes mandated by Gumboot", which would achieve the same but not require accession and the related structure of roles/permissions.

Comment (smangelsdorf) [inline doc/layout_helpers.md]: Better?

**Comment by rianniello** [comment]
All looks good to me!

**Pull request merged**